### PR TITLE
feat(adoption-v2): decision about adoption

### PR DESCRIPTION
**Pull request by paulgrym**
## Jira issue

JIRA issue: [FSU-249](https://klau.atlassian.net/browse/FSU-249)

## Describe your changes
This PR introduces Decision About Adoption section for Adoption Page v2.

## Checklist before requesting a review

- [x] Checked which branch you would merge.
- [x] Checked if the code follows the style guidelines of this project (use eslint and prettier).
- [x] Performed a self-review of code (bugs, clean code rules, etc).
- [x] Checked if added variables have unique names.
- [x] Checked if added code/function is not a duplicate. 
- [x] Tests have been added if possible.
- [x] New and existing unit tests pass locally with my changes.

### CODE REVIEW

File: packages/frontend/app/templates/Adoption/components/AdoptionDecision/AdoptionDecision.jsx [hunk 1]
```diff
@@ -0,0 +1,143 @@
...
+import Typography from "design-system/components/typography"
+import globalData from "shared/data"
+import * as Styled from "./AdoptionDecision.styled"
+
+const AdoptionDecision = () => {
+  const theme = useTheme()
+
+  const accordions = [
+    {
+      heading: "Zastanów się nad adopcją wspólnie z każdym członkiem rodziny",
```
Comment (AmandaKr) [inline packages/frontend/app/templates/Adoption/components/AdoptionDecision/AdoptionDecision.jsx:14]: I know this is not your change, but the accordion heading and arrow are not aligned. My suggestion is to add `justify-content: center` to Accordion in Accordion.styled.js
image

**Review by AmandaKr — APPROVED** [review]
🚀

File: packages/frontend/app/templates/Adoption/components/AdoptionDecision/AdoptionDecision.jsx [hunk 2]
```diff
@@ -0,0 +1,143 @@
...
+      heading: "Wybierz psa i dowiedz się o nim jak najwięcej",
+      text: "Jeśli jakiś pies Cię zainteresował, przeczytaj uważnie jego opis na naszej stronie, aby dowiedzieć się jakie ma potrzeby, temperament oraz jakie jest jego zachowanie. ",
+      icon: "decisionPaw",
+    },
+    {
+      heading: "Poznaj psa, zanim go adoptujesz",
+      text: "Poznaj wybranego psa na spotkaniu przedadopcyjnym. Spotkaj się z nim i spędź trochę czasu. To pomoże ocenić Ci, czy pasujecie do siebie i czy czujesz z nim więź. Na takim spotkaniu również Fundacja będzie miała okazję poznać bliżej Ciebie.",
+      textBottom:
+        "W tym celu wypełnij ankietę przedadopcyjną i odeślij do nas:",
+      button: {
```
Comment (im-klau) [inline packages/frontend/app/templates/Adoption/components/AdoptionDecision/AdoptionDecision.jsx:62]: this is rendered as `button` not `<a>` element which means the href isn't working 🙈

File: packages/frontend/app/templates/Adoption/components/AdoptionDecision/AdoptionDecision.jsx [hunk 3]
```diff
@@ -0,0 +1,143 @@
...
+        text: "Ankieta przedadopcyjna",
+        variant: "text",
+        color: "primary",
+        icon: "download",
+        size: "medium",
+        href: `${globalData.surveys.preAdoption}`,
```
Comment (im-klau) [inline packages/frontend/app/templates/Adoption/components/AdoptionDecision/AdoptionDecision.jsx:68]:
```suggestion
        href: globalData.surveys.preAdoption,
```

File: packages/frontend/app/templates/Adoption/components/AdoptionDecision/AdoptionDecision.jsx [hunk 4]
```diff
@@ -0,0 +1,143 @@
...
+import Icon from "design-system/components/icon"
+import Button from "design-system/components/button"
+import Typography from "design-system/components/typography"
+import globalData from "shared/data"
+import * as Styled from "./AdoptionDecision.styled"
+
+const AdoptionDecision = () => {
+  const theme = useTheme()
+
+  const accordions = [
```
Comment (im-klau) [inline packages/frontend/app/templates/Adoption/components/AdoptionDecision/AdoptionDecision.jsx:12]: what a nice data object! ✨

File: packages/frontend/app/templates/Adoption/components/AdoptionDecision/AdoptionDecision.styled.js [hunk 5]
```diff
@@ -0,0 +1,79 @@
...
+`
+export const AccordionListItem = styled(ListItem)`
+  align-items: flex-start;
+  gap: 0;
+
+  svg {
+    width: 28px;
+    height: 28px;
+  }
+`
```
Comment (im-klau) [inline packages/frontend/app/templates/Adoption/components/AdoptionDecision/AdoptionDecision.styled.js:58]: Nice workaround to have this kind of list completed! 🚀

**Pull request merged**